foundation/foundation-upgrade/src/test/resources/specifications/v1_7_0/data-lineage-migration.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Data Lineage upgrade migration
+
+  Scenario: Update Data Lineage package name
+    Given a Java file references the package com.boozallen.aiops.data.lineage
+    When the one.seven.zero data lineage package migration executes
```

Comment:
Q/A: Is this spelled out because the plugin that generates the glue code methods was treating the numbers as params? If so, would recommend just manually changing it to `the 1.7 data...` in the `@When` so this is a bit more readable.